jupyter/rocm/pytorch/ubi9-python-3.9/de-vendor-torch.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+set -ex
+# Replace PyTorch's vendored shared libraries with system libraries
+# The script assumes that PyTorch is built with the same ROCm ABI as the
+# system installation of ROCm.
```

Comment:
With this, I suppose we need to be careful for these two to be in line:
* https://github.com/opendatahub-io/notebooks/blob/daab9530693853b4883ea4a5fe39a3e2ae2c344a/jupyter/rocm/pytorch/ubi9-python-3.9/Pipfile#L16
   * https://github.com/opendatahub-io/notebooks/blob/daab9530693853b4883ea4a5fe39a3e2ae2c344a/jupyter/rocm/pytorch/ubi9-python-3.9/Pipfile#L7
* https://github.com/opendatahub-io/notebooks/blob/daab9530693853b4883ea4a5fe39a3e2ae2c344a/rocm/ubi9-python-3.9/Dockerfile#L18

But we should be careful for these to be in line also without it anyway.

BTW, I can see a discrepancy there ROCm 6.0 vs 6.1 - shall we sync it @harshad16 ?

Reply:
> BTW, I can see a discrepancy there ROCm 6.0 vs 6.1 - shall we sync it @harshad16 ?

seen this before but I guess I forgot, thanks for bringing it up!

Reply:
Back when we developed, pytorch 6.1 was not available, it seems to be available fairly recently.

Reply:
Okay, I raised #656 to address this. Feel free to review and comment there. Thank you!

Reply:
It's resolved now, so let's proceed here

* https://github.com/opendatahub-io/notebooks/pull/656#event-13737876179